curriculum/challenges/english/05-apis-and-microservices/apis-and-microservices-projects/url-shortener-microservice.english.md

```diff
@@ -24,13 +24,81 @@ Start this project on Glitch using <a href='https://glitch.com/edit/#!/remix/clo
 
 ```yml
 tests:
+  - text: I can provide my own project, not the example URL.
```

Comment:
```suggestion
You should provide your own project, not the example URL.
```

Reply:
@RandellDawson  updated.

Reply:
Maybe I should update the wording in the other available (already merged) tests in the category of apis-and-microservices such as `exercise-tracker`, "timestamp-microservice" file so they are consistent. The wording in the `request-header-parser-microservice`test looks ok. I can do that in another PR to not mix the scope of this one.

Reply:
@abbathaw, sure, that sounds like a great idea. Go ahead and update that text in another PR whenever you get a chance.